src/Symfony/Component/HttpKernel/Tests/Controller/ControllerResolverTest.php

```diff
@@ -196,6 +197,22 @@ public function testGetArguments()
         $this->assertEquals(array($request), $resolver->getArguments($request, $controller), '->getArguments() injects the request');
     }
 
+    /**
+     * @requires PHP 5.6
+     */
+    public function testGetVariadicArguments()
+    {
+        $resolver = new ControllerResolver();
+
+        $request = Request::create('/');
+        $param1 = new \stdClass();
+        $param2 = new \stdClass();
+        $request->attributes->set('foo', 'foo');
+        $request->attributes->set('bar', array($param1, $param2));
+        $controller = array(new VariadicController(), 'action');
+        $this->assertEquals(array('foo', $param1, $param2), $resolver->getArguments($request, $controller));
```

Comment:
Should be `assertSame()` as  `assertEquals()` would not make a distinction between `['foo', $param1, $param2]` and `['foo', $param1, $param1]`. 

Also, consider using strings instead of `stdClass` here. In this case type doesn't matter and if we can make the test shorter, we should ;)